packages/tupaia-web-server/src/routes/MapOverlaysRoute.ts

```diff
@@ -57,6 +58,7 @@ export class MapOverlaysRoute extends Route<MapOverlaysRequest> {
           comparisonValue: [projectCode],
         },
       },
+      pageSize: PAGE_SIZE,
```

Comment:
Should pageSize be sent from the client instead? Or is the client not pagination aware when it should be?

I'm not super across this epic branch but seems like we will have the same issue with all REST endpoints, and it might be better to do a generic solution than a specific one.

Reply:
Yeah usually I just pass through the user's query params to this main fetch, but in a case like this where we're doing multiple fetches for relationship entries, the later fetches may have more entries than this initial one. So we can't take `pageSize` from the user for all of these.

I've updated to pass `pageSize: 'ALL'` which will remove the `100` restriction that the `central-server` uses, but also the allow the user to override that with their query

Reply:
Ah ok sounds good!